src/clientdlg.cpp

```diff
@@ -277,7 +277,13 @@ CClientDlg::CClientDlg ( CClient*         pNCliP,
     {
         // initiate connection (always show the address in the mixer board
         // (no alias))
-        Connect ( strConnOnStartupAddress, strConnOnStartupAddress );
+
```

Comment:
I don't think ClientDlg should be responsible for any of this.

Client should handle it.  It should get passed a ClientDlg instance (or null if headless), initialise the ClientDlg (if present), initialise audio and then initiate any "on start up" server connection.  That should be enough to get everything to "just work".

Reply:
Changed my mind a bit about this...

That's the problem with "small changes" that affect the way the application works.  They're not really "small".